### PR TITLE
Import assignment docs

### PR DESCRIPTION
**Pull request by PradyumnaShome**

### CODE REVIEW

**Comment by Xiangmingchen** [comment]
Is there any change besides moving the files?

File: README.MD [hunk 1]
```diff
@@ -0,0 +1,111 @@
+# Webpage for CS 241
```
[the remark this answers is not in the document]
Comment (PradyumnaShome) [reply README.MD]: Good catch!

**Comment by PradyumnaShome** [comment]
Nope that was it.

**Comment by Xiangmingchen** [comment]
Wait, my jekyll can't compile. Here is the error message: 

> Liquid Exception: Could not find document '_coursebook/Background.md' in tag 'link'. Make sure the document exists and the path is correct. in /Users/jenny/gitRepos/illinois-cs241.github.io/_docs/lockpicking_lab.md

**Comment by Xiangmingchen** [comment]
Never mind, seems like I didn't follow the set up correctly.

**Comment by ASankaran** [comment]
I think this should work. But to verify can you (@PradyumnaShome) trigger a travis build without the deploy step to ensure that travis can build it corretly.

**Comment by PradyumnaShome** [comment]
Succeeded: https://travis-ci.org/illinois-cs241/illinois-cs241.github.io/jobs/651173695

**Pull request merged**